index.html

```diff
@@ -1285,4 +1286,4 @@ <h2 id="related-projects">Related Projects<a class="headerlink" href="#related-p
       
     
   </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Not that I intended it, but GitHub's own editor (in a browser) automatically adds it.